ci-bootstrap/terraform/role.tf.tmpl

```diff
@@ -0,0 +1,87 @@
+# Run via ci-bootstrap/bin/deploy.sh
+
+provider "aws" {
+    region = "eu-west-2"
+}
+
+variable "environment" {}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_iam_role" "deploy-mail-lambda" {
+  name = "bpm-svc-deploy-mail-lambda"
+  assume_role_policy = data.aws_iam_policy_document.instance-assume-role-policy.json
+}
+
+resource "aws_iam_role_policy" "deploy-mail-lambda-inline" {
+  name = "DeployMailLambdaPolicy"
+  role = aws_iam_role.deploy-mail-lambda.name
+  policy = data.aws_iam_policy_document.bpm-deploy-lambda-policy.json
+}
+
+data "aws_iam_policy_document" "instance-assume-role-policy" {
+  statement {
+    sid = "AssumeDeployRole"
+    actions = [
+      "sts:AssumeRole"
+    ]
+    principals {
+      type = "AWS"
+      identifiers = [
+        "arn:aws:iam::736089984042:user/bpm-svc-concourse"
```

Comment:
@necrophonic what do you think?

Reply:
I'm not totally against it (I mean this was affectionately ripped off from my code ;)). Ideally it could be variabled out?

Ideally though, this repo should be private as it contains infrastructure.